api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/SaveBookmarkRequest.java

```diff
@@ -0,0 +1,49 @@
+package org.motechproject.nms.api.web.contract.mobileAcademy;
+
+import java.util.Map;
+
+/**
+ * Bookmark request object to save a bookmark 2.2.5
+ */
+public class SaveBookmarkRequest {
+
+    private Long callingNumber;
+
+    private Long callId;
+
+    private String bookmark;
+
+    private Map<String, Integer> scoresByChapter;
+
```

Comment:
Interesting. The default argument-less constructor gets auto-generated *if* no other constructor is defined, which explains why my IT for this didnt fail. I will add the constructor explicitly to keep up with the standard though.